### ISSUE TITLE
Zeus losing passive with unique attack modifier

### ISSUE THREAD

**Issue by Tesstreams** [ValveSoftware/Dota2-Gameplay#8648]
### Ability name

Lightning hands

### Description

in this example, Zeus loses lightning hands after equipping eye of skadi.

### Example Match ID (and possibly Timestamp)

7125760841 46:30

**Comment by TheUnholyWrath** [comment]
Turning off the auto cast on Lightning Hands disables the ability. If you think that this is working incorrectly you should make a new issue regarding its functionality or remake this one. Otherwise, you should close this issue.

**Comment by Tesstreams** [comment]
It wasn't about auto cast. Does it, or does it not override the lightning
having skadi? I said nothing about the alt toggle

**Comment by TheUnholyWrath** [comment]
> It wasn't about auto cast. Does it, or does it not override the lightning having skadi? I said nothing about the alt toggle

Yeah I'm sorry thats my mistake, I assumed the Zeus player was the one reporting this. Watching the replay we know it can't possibly be Skadi causing this. After assembling Skadi, Zeus can be seen farming an Ancient camp at 35:01 where Lightning Hands is working correctly. 
![20230426180351_1](https://user-images.githubusercontent.com/46859977/234513445-c410e635-f72b-4a9d-accc-58470bf2e730.jpg)

Lightning Hands is last seen working at 36:19 while Zeus is farming a camp of Mud Golems. Zeus toggles off Lightning Hands at 36:36. 
![20230426180237_1](https://user-images.githubusercontent.com/46859977/234513902-6aa585d0-fd26-4cd6-91d0-89c09840b083.jpg)

Lightning Hands doesn't work non-illusion versions of Zeus for the rest of the game. Zeus also later in the game tests to see if Skadi is the thing causing this issue, and Lightning Hands still doesn't work. 

![20230426180809_1](https://user-images.githubusercontent.com/46859977/234514400-e08c7b41-4e92-44ae-b62a-21bf769b1e90.jpg)

This is all also replicable in demo

https://user-images.githubusercontent.com/46859977/234514881-38f948f4-4e5d-4cf5-84ee-a90249651567.mp4

**Comment by anotherCoward** [comment]
Some Players seem to be confused that it is an Auto-Cast Ability and not a passive and that they can disable it by accident.

There is another issue #8541 with the same behavior. 

Some players seem to require a setting like "Auto-Cast can only be toggled with keyboard shortcut".

**Comment by Tesstreams** [comment]
Ok, thanks for the clarification! Yeah it was happening during a
Livestream, and like around 46 minutes was where either user error
happened, or weird bug behavior. Everyone was confused for a bit lmao
